Dockerfile

```diff
@@ -28,7 +28,7 @@ RUN apk update && apk add --no-cache \
   postgresql15-client \
   redis \
   nginx && \
-  if [ "$TARGETOS" = "linux" ] && [ "$TARGETARCH" != "s390x" ]; then \
+  if [ "$TARGETOS" = "linux" ] && [ "$TARGETARCH" != "s390x" ] && [ "$TARGETARCH" != "ppc64le" ]; then \
```

Comment:
Hey @KaushikOP The `if` condition change looks correct. @hash-d what do you think?